cookbook/core/control_flow/dynamics.py

```diff
@@ -1,14 +1,12 @@
 """
-Dynamic Tasks
---------------
+Dynamic Workflows
```

Comment:
I thought we wanted tasks.  https://github.com/flyteorg/flytesnacks/pull/291/files

Reply:
I thought so too, but throughout the docs (on the page itself too), and the [blog post](https://blog.flyte.org/dynamic-workflows-in-flyte) we reference "dynamic workflows". @kumare3 any input?